src/main/java/use_case/recipeCollection/RecipeCollectionInteractor.java

```diff
@@ -0,0 +1,33 @@
+package use_case.recipeCollection;
+
+import java.util.List;
+
+import entity.Recipe;
+
+/**
+ * The Recipe Collection interactor.
```

Comment:
Already having Javadocs is best practice, great work on this! What I would say as feedback is to provide detail regarding what a method and/or class and other objects do. This will improve readability not only for you but for other team members working on the code as well! For instance, in this scenario, writing what the interactor does and what objects they use will improve understanding and readability.